nix/overlays.nix

```diff
@@ -0,0 +1,4 @@
+{ pkgs, packages }:
```

Comment:
Having the overlay depend on Nixpkgs is a bit weird (since any nixpkgs version used should be passed in via `super`).
Since it's not used, better not have it at all.

(I'd also suggest that `genealogos-api` in the overlay should be built against `super` because that's what people would expect when using the overlay. But that can be saved for a follow-up)